sample/EasyTables/CSharp/QueueToInItemToOutItem/function.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "samples-workitems"
+    },
+    {
+      "type": "easyTable",
+      "name": "queriedItem",
+      "direction": "in",
+      "id": "{QueueTrigger}",
+      "tableName": "Item"
+    },
+    {
+      "type": "easyTable",
```

Comment:
This binding confuses me. No table name is given. What is it binding to? Is the item new or modified?